docs/security-model.mdx

```diff
@@ -11,11 +11,11 @@ To achieve that, Ory manages:
 - tokens
 - cookies
 
-While this is not a common approach that might require a shift in thinking and some getting used to, it's implemented so that you
+Although it is an uncommon approach and might require a shift in thinking, Ory has implemented it so that you
```

Comment:
```suggestion
Although it is an uncommon approach that might require a shift in thinking, it's implemented so that you
```

I'd like to limit the use of "Ory" in writing where possible. We're in the documentation for Ory after all, so no need to use the company that often :)